.github/actions/validate-issue/action.yml

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2023, salesforce.com, inc.
+# All rights reserved.
+# Licensed under the BSD 3-Clause license.
+# For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+#
+
+name: "Salesforce CLI Validate Issues"
+description: "Validate information provided in an new Github issue."
+author: "Eric Willhoit"
+inputs:
+  repo-token:
+    required: true
+    description: "Token taken from secrets env var"
+runs:
+  using: "node16"
```

Comment:
I'd go higher since 16 is EOL in October.

I couldn't tell from the docs if there's a way to specify lts, but that would be best.

Reply:
It's not supported yet 😑 https://github.com/actions/runner-images/issues/7002#issuecomment-1470047544
I tried a work around there but it did not seem to work. Going to wait it out and hopefully they will add support